packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -165,6 +161,17 @@ export class TableRow<
         }
     }
 
+    public handleChange(source: unknown, args: unknown): void {
```

Comment:
Mark `internal`?

Reply:
Yes, this is internal. We hadn't marked any of the internal functions as such in this file, so I went ahead and put that tag on all that it applies to.